src/transformers/models/auto/processing_auto.py

```diff
@@ -144,47 +138,28 @@ def from_pretrained(cls, pretrained_model_name_or_path, **kwargs):
         config = kwargs.pop("config", None)
         kwargs["_from_auto"] = True
 
-        # First, let's see if we have a preprocessor config.
-        # get_list_of_files only takes three of the kwargs we have, so we filter them.
-        get_list_of_files_kwargs = {
-            key: kwargs[key] for key in ["revision", "use_auth_token", "local_files_only"] if key in kwargs
-        }
-        model_files = get_list_of_files(pretrained_model_name_or_path, **get_list_of_files_kwargs)
-        if FEATURE_EXTRACTOR_NAME in model_files:
-            config_dict, _ = FeatureExtractionMixin.get_feature_extractor_dict(pretrained_model_name_or_path, **kwargs)
-            if "processor_class" in config_dict:
-                processor_class = processor_class_from_name(config_dict["processor_class"])
-                return processor_class.from_pretrained(pretrained_model_name_or_path, **kwargs)
```

Comment:
I think it's cleaner to leave this here and only check the feature extractor config for a processor class if there is one available. There might be processors in the future that require no feature extractor.